_posts/en/posts/2017-02-24-release-0.14.0.md

```diff
@@ -0,0 +1,134 @@
+
+The existing signature cache in Bitcoin Core 0.13.2 and below works well for systems with fewer than eight CPU cores, but for systems with eight or more CPU cores a bottleneck (lock contention) appears that prevents the system from using all available cores as effectively as possible.  The upgrade to using cuckoo hashing to provide a "cuckoo cache" eliminates this problem and allows more cores to be used effectively.
+
+In a [test][morcos test] on a system with 16 cores, a new block was able to be added (connected) to the block chain 40% faster using 0.14.0 than a previous version of Bitcoin Core.  For systems with fewer than 8 cores, there is no major performance increase, although the cuckoo cache does allow caching more signatures than before for the same amount of memory, so there can be a slight performance improvement.
+
+### High-bandwidth BIP152 compactblock implementation
+
+The second feature improved in Bitcoin Core 0.14.0 is the BIP152 compactblock implementation.  The previous implementation supported BIP152's low-bandwidth mode; the upgraded implementation now supports the high-bandwidth mode.
+
+![Compact blocks protocol flow](https://raw.githubusercontent.com/bitcoin/bips/master/bip-0152/protocol-flow.png)
+
+The new high-bandwidth mode uses the same efficient data transfer methods as the previous low-bandwidth mode but it is willing to relay new block data that has not been fully validated and also immediately send other data the receiving node might need without waiting for the receiving node to first confirm it needs that data.  This risks relaying invalid and redundant data---but provides the benefit of ensuring that data which could be valid is relayed as fast as possible.
+
+In the best case, removing the validation delay can allow new blocks to propagate across multiple hops on the peer-to-peer network several times faster than they could before.  In the worst case, some bandwidth is wasted, but the security model is not changed since all nodes will still reject invalid blocks.
+
+Bitcoin Core 0.14.0 nodes using BIP152 high-bandwidth mode remain compatible with Bitcoin Core 0.13.x and earlier nodes by using protocol negotiation to ensure they only enable high-bandwidth mode with peers who also support it.
+
+### P2P code refactor focused on concurrency and throughput
+
+The third feature improved in Bitcoin Core 0.14.0 is the entire set of P2P networking code, which has been refactored to support increased concurrency and throughput.  The concurrency improvements help allow newly-received blocks (such as BIP152 compactblocks) to be processed ahead of lower-priority traffic, ensuring that blocks are relayed and validated as fast as possible.
+
+The refactor also now allows network activity to continue on spare CPU processors during validation, providing an improvement in IBD speed that complements the [headers-first sync][] introduced in Bitcoin Core 0.10.0 (see [above][ibd] for more information).
```

Comment:
s/on spare CPU processors/in the background/. Even single-core CPUs benefit here.

s/during validation, providing/during message processing, notably providing/. Latency for all message handling is reduced, it's just most apparent during IBD.